src/core/ticker/index.js

```diff
@@ -53,6 +53,10 @@ shared.destroy = () =>
 };
 
 /**
+ * This namespaces contains an API for interacting with PIXI's internal global loop.
```

Comment:
'namespace', singular. It might just be me, but I feel like there should be another word between global and loop, like 'global update loop'. Then again, that sounds like a little more of a mouthful than it should be.